com.unity.render-pipelines.high-definition-config/Runtime/ShaderConfig.cs

```diff
@@ -36,6 +42,13 @@ public enum ShaderOptions
 
         /// <summary>Support to apply a global mip bias on all texture samplers of HDRP.</summary>
         GlobalMipBias = 1,
+
+        /// <summary>
+        /// Maximum number of lights for a fine pruned light tile. This number can only be 3 prespecified possibilities, 23, 31 or 63.
+        /// The reserved index is there for the count of the fined prunned tile list. The lower the count, some memory savings will occur for the tile light list.
+        /// Note: For any rendering bigger than 4k (in native) it is recommended to use Medium or Small light counts per tile, to avoid possible artifacts.
```

Comment:
This is confusing. In the doc there is mention of two possible values (32 & 64), but here you're mentioning 3 and recommend using 23 or 31 in big resolutions. And the Enum only has two. What is the right information ?